### PR TITLE
prepare repo-metadata.json for docuploader

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**
name should match DNS name of API service
distribution_name should be the gem name

### CODE REVIEW

File: google-cloud-asset/.repo-metadata.json [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
-  "distribution_name": "google-cloud-asset",
-  "module_name": "Asset",
-  "module_name_credentials": "Asset::V1",
-  "env_var_prefix": "ASSET"
-}
\ No newline at end of file
+    "name": "cloudasset",
```
Comment (blowmage) [inline google-cloud-asset/.repo-metadata.json]: 4 spaces?!?! 😆

File: Rakefile [hunk 2]
```diff
@@ -815,4 +815,16 @@ def extract_args args, *keys
   vals.length > 1 ? vals : vals.first
 end
 
+task :what do
+  gems.each do |gem|
+    filename = "#{gem}/.repo-metadata.json"
+    if File.file? filename
+      puts filename
+      content = File.read filename
+      content.gsub! "    ", "  "
```
Comment (blowmage) [inline Rakefile]: Noooo, I was kidding! :rofl:

Comment (blowmage) [reply Rakefile]: I love the commit title tho!

Comment (TheRoyalTnetennba) [reply Rakefile]: Sometimes, I don't know why I even try

**Pull request merged**